ThunderCloud/Quiz.swift

```diff
@@ -37,10 +46,12 @@ public struct QuizConfiguration {
     ///   - shuffleQuestions: Whether quiz questions should be shuffled
     ///   - requireAnswer: Whether answers are required before progressing to next question
     ///   - isBlendedLearningEnabled: Whether blended learning features are enabled
-    public init(shuffleQuestions: Bool = false, requireAnswer: Bool = true, isBlendedLearningEnabled: Bool = false) {
+    ///   - maxNumberOfQuestions: The max number of questions for the quizz
+    public init(shuffleQuestions: Bool = false, requireAnswer: Bool = true, isBlendedLearningEnabled: Bool = false, maxNumberOfQuestions: Int) {
```

Comment:
This parameter should be `Int?` and we need to default it to `nil` so it's not a breaking change